server/src/test/java/keywhiz/service/resources/admin/SecretsResourceTest.java

```diff
@@ -310,9 +310,7 @@ public void deleteErrorsOnNotFound() {
   }
 
   @Test(expected = ConflictException.class)
-  public void triesToCreateDuplicateSecret() throws Exception {
-    SecretController.SecretBuilder secretBuilder = mock(SecretController.SecretBuilder.class);
-    when(secretController.builder("name", "content", user.getName(), 0)).thenReturn(secretBuilder);
+  public void triesToCreateDuplicateSecret() {
     DataAccessException exception = new DataAccessException("");
     doThrow(exception).when(secretBuilder).create();
```

Comment:
Unrelated cleanup, but: this test did not need to declare any thrown exception (so I removed it) and it was duplicating some builder mocking which was being done in the setup method (also removed).

Reply:
I think we have a lot of unit tests unnecessarily throwing an exception...

Reply:
Sometimes I'd just slap "throws Exception" onto tests just to prevent any compilation issues around thrown exceptions, especially as test change over time, but IDEs also tend to nag about unnecessary throws declarations (like this one) so I figure getting rid of nag highlights is a win for reducing cognitive load. One less thing to worry about.